gufe/tests/conftest.py

```diff
@@ -67,7 +67,12 @@ def get_test_filename(filename):
     name: URLFileLike(url=_pl_benchmark_url_pattern.format(name=name)) for name in _benchmark_pdb_names
```

Comment:
Need to hard-code release version here (or better, just vendor all the PDBs into gufe)